src/primitives.jl

```diff
@@ -179,6 +181,27 @@ end
 dropnull(X::NullableVector) = X.values[!X.isnull]    # -> Vector
 
 """
+    dropnull!(X::AbstractVector)
+
+Remove null entries of `X` in-place and return the updated vector. This is a
+no-op if no nulls are present.
+
+    dropnull!(X::NullableVector)
```

Comment:
is the preferred style to use 4 space indent like I did here or to use backticks as used [here](https://github.com/JuliaStats/NullableArrays.jl/blob/master/src/primitives.jl#L237)